quickwit-directories/Cargo.toml

```diff
@@ -19,7 +19,7 @@ quickwit-storage = { version = "0.3.1", path = "../quickwit-storage" }
 serde = "1"
 serde_cbor = "0.11"
 serde_json = "1"
-tantivy = { git = "https://github.com/quickwit-oss/tantivy/", rev = "d24f31f", default-features = false, features = [
+tantivy = { git = "https://github.com/quickwit-oss/tantivy/", rev = "f4d7621", default-features = false, features = [
```

Comment:
Try to make a separate PR to update tantivy in the future, so it's clear which changes are required by the tantivy upgrade